options.go

```diff
@@ -224,6 +231,9 @@ type Options struct {
 	// will try to stop containers created by it right after the tests exit.
 	DisableAutoCleanup bool `json:"disable_cleanup"`
 
+	// UseLocalImagesFirst to avoid hitting the docker hub pull rate limit.
+	UseLocalImagesFirst bool `json:"use_local_images_first"`
```

Comment:
This option should be added to [Swagger spec](https://github.com/orlangure/gnomock/blob/master/swagger/swagger.yaml#L441) as well for non-Go usages.